common.yml

```diff
@@ -690,6 +717,11 @@ timeout:
           # git the processes a second or two to dump their stacks
           sleep 10
         fi
+  # Extra timeout steps needed for mongodump passthrough.
+  - func: f_expansions_write
+  # Hang analyzer is used to upload data files after a timed-out resmoke test.
+  - func: "run hang analyzer"
```

Comment:
I'm not sure we ever actually run this for Mongosync, because for some reason we _can't_. So I'm not sure if this is useful here either.

Reply:
I'm slightly inclined to match what mongosync evergreen has as closely as possible.  But if you're 100% sure it's useless, I could take it out.

Reply:
Eh, it's fine either way, I think.

Reply:
I removed it.